### DISCUSSION TITLE
Rate limit issues

### DISCUSSION THREAD

**Discussion by haringsrob** [pkuehnel/TeslaSolarCharger#1313] [General]
It looks like I hit the rate limit rather quick.

I am thinking about setting up a BLE raspberry. But I wonder how close it needs to be? is 10meters enough?

Cheers.

**Comment by pkuehnel** [comment]
I have no idea, just try.

**Reply by my7ho5** [reply]
Totally depends on what's between the distance. For me 10 meters with 2-3 walls (linear vs reflection) works fine.

**Reply by haringsrob** [reply]
Thanks for the input!
